geode-management/src/main/java/org/apache/geode/management/api/ConfigurationResult.java

```diff
@@ -46,7 +51,12 @@ public ConfigurationResult() {}
    * for internal use only
    */
   public ConfigurationResult(T configuration) {
+    this(configuration, emptyList());
+  }
+
+  public ConfigurationResult(T configuration, List<R> runtimeInfo) {
```

Comment:
It seems like a better name for this class would be "GroupResult". I like that you updated the class comment to say it is per group. It seems like this class should include a group name so that it would be self describing.